src/main/dml/qubAccessControl.dml

```diff
@@ -8,7 +8,8 @@ class AccessControlProfile  {
  String rawName (REQUIRED) ; 
  String code  (REQUIRED) ;
  Boolean restricted (REQUIRED) ; 
- Boolean locked (REQUIRED) ; 
+ Boolean locked ; // Deprecated
+ Boolean system ;
```

Comment:
@francisco-esteves shouldn't System be required? Since locked was and we are migrating from locked to system? This will be validated during the TXs, as long as you perform a migration should be fine.

Reply:
I didn't set system to required because I was unsure if a transaction involving a profile in the access control manager's permission initializer would run before the migration in the artifact handler, leading to an exception being thrown.
But I haven't tested this theory yet.

Reply:
Very well thought. Let's keep it without being required for now and after migration we release a new version where system is REQUIRED again.